plugins/parsers/json/parser.go

```diff
@@ -32,6 +32,7 @@ type Config struct {
 	Timezone     string
 	DefaultTags  map[string]string
 	Strict       bool
+	Escape       bool
```

Comment:
I'm not sure `Escape` is a "good" name. How about `Quoted` referencing the call to `strconv.Unquote`? Just a thought...